applications/nrf5340_audio/src/bluetooth/le_audio_cis_headset.c

```diff
@@ -467,12 +467,9 @@ static void stream_recv_cb(struct bt_bap_stream *stream, const struct bt_iso_rec
 
 static void stream_start_cb(struct bt_bap_stream *stream)
 {
-	int ret;
-
 	LOG_INF("Stream %p started", stream);
```

Comment:
Then we might need to make the log level consistent between stream_stop and stream_start.

Reply:
Done